packages/xo-server/src/api/proxy.js

```diff
@@ -66,3 +66,21 @@ update.params = {
     optional: true,
   },
 }
+
+export function call({ id, method, params }) {
```

Comment:
This should only be used for testing and not commited to the final version, please add a comment explaining this.

Reply:
If not necessary, I think we should remove now or we'll forget to do it.